src/sentry/projects/services/project/model.py

```diff
@@ -54,6 +84,11 @@ def delete_option(self, key: str) -> None:
 
         project_service.delete_option(self, key)
 
+    def get_flags(self, organization_id: int, project_id: int) -> RpcProjectFlags:
+        from sentry.projects.services.project import project_service
+
+        return project_service.get_flags(organization_id=organization_id, project_id=project_id)
```

Comment:
Should flags be part of `RpcProject` all the time? There are relatively few read operations to projects over RPC so adding to the response payload should have minimal impact. If you wanted to be even more cautious, you could add parameters to the existing get methods to include flags in the results.

Reply:
As you suggested in getsentry, it may actually be more useful to move this to RpcOrganization instead and combine flags in that call - and that obviates the need for a permanent representation, especially if we allow filtering by a subset of the project ids in the method.